ncm-opennebula/src/main/pan/components/opennebula/schema.pan

```diff
@@ -147,11 +161,14 @@ type opennebula_datastore = {
 
 type opennebula_vnet = {
     "name" : string
-    "type" : string  = 'FIXED'
```

Comment:
this is backwards incompatible change.
@jrha since there is no release with the opennebula component, i guess it's ok-ish to do so?